canvas_modules/common-canvas/src/palette/palette.scss

```diff
@@ -217,8 +220,10 @@ $palette-border-color: $ui-03;
 
 .palette-flyout-category-item-icon {
 	width: 20px;  /* Not specifying height preserves the image's aspect ratio. */
-	margin-left: 16px;
+	margin-left: 5px;
 	fill: $icon-01;
+	padding-top: 0rem;
+	margin-right: 0.2rem;
```

Comment:
We generally use pixel (px) sizes in the SCSS rather than rem units. Is there some reason you are using rem here? Can you use px here if possible please?